src/pull_automerge/core.clj

```diff
@@ -89,23 +89,40 @@
   (println "Get statuses result:" (result :status))
   result)
 
+;Get all objects in response data that contains the specified label
+(defn filter-by-label [ response-data label ]
+  (filter 
+    #(> (count (filter (fn[m] (= (m :name) label)) (% :labels))) 0) 
+    (json/read-str (response-data :body) :key-fn keyword)))
+
+;Get entry from list with the earliest creation date
```

Comment:
Probably don't need the comments, just stuck them in there so it was more obvious what they were doing when you were reviewing. I'll remove these as well.